packages/compose/src/hooks/use-ref-effect/index.js

```diff
@@ -17,14 +17,15 @@ import { useCallback, useRef } from '@wordpress/element';
  * to be removed. It *is* necessary if you add dependencies because the ref
  * callback will be called multiple times for the same node.
  *
- * @param {Function} callback     Callback with ref as argument.
- * @param {Array}    dependencies Dependencies of the callback.
+ * @param {(node: Node) => (() => void) | undefined} callback     Callback with ref as argument.
+ * @param {import('react').DependencyList}    dependencies Dependencies of the callback.
```

Comment:
The tab formatting looks off — in the first line the tab is _after_ the param name, in the second line the tab is _before_ the param name